vignettes/getting-started.Rmd

```diff
@@ -19,24 +19,24 @@ knitr::opts_chunk$set(
 gimap performs analysis of dual-targeting CRISPR screening data, with the goal of aiding the identification of genetic interactions (e.g. cooperativity, synthetic lethality) in models of disease and other biological contexts. gimap analyzes functional genomic data generated by the pgPEN (paired guide RNAs for genetic interaction mapping) approach, quantifying growth effects of single and paired gene knockouts upon application of a CRISPR library. A multitude of CRISPR screen types can be used for this analysis, with helpful descriptions found in this review (https://www.nature.com/articles/s43586-021-00093-4). Use of pgPEN and GI-mapping in a paired gRNA format can be found here (https://pubmed.ncbi.nlm.nih.gov/34469736/). 
 
 ```{r}
-library(magrittr)
+library(dplyr)
 library(gimap)
 ```
 
 ## Data requirements 
 
 Let's examine this example pgPEN counts table. It's divided into columns containing: 
 
-- an ID corresponding to the names of paired guides
-- gRNA sequence 1, targeting "paralog A"
-- gRNA sequence 2, targeting "paralog B"
-- The sample, day, and replicate number for which gRNAs were sequenced
+- `id`: an ID corresponding to the names of paired guides
+- `seq_1`: gRNA sequence 1, targeting "paralog A"
+- `seq_2`: gRNA sequence 2, targeting "paralog B"
+- The sample, day, and replicate number for which gRNAs were sequenced (TODO)
```

Comment:
I was unclear what the last bullet point was saying. The column names in `example_data` that this point is referencing, `Day00_RepA`, `Day05_RepA`, `Day22_RepA` `Day22_RepB` `Day22_RepC`, doesn't seem to have any info on the sample. I think  the names refer to the Day, and Replicate number, but not the sample. 

I would list out each column and describe what each column is. For ex,

- `Day00_RepA`: Gene Counts from Day 00 for Replicate A 
- `Day05_RepA`: Gene Counts from Day 05 for Replicate A 
- `Day22_RepA`: Gene Counts from Day 22 for Replicate A 
- `Day22_RepB`: Gene Counts from Day 22 for Replicate B

Reply:
This also sounds great! Can you add this?